tests/console/check_os_release.pm

```diff
@@ -62,6 +62,15 @@ sub run {
         $checker{PRETTY_NAME} = $checker{NAME};
     }
 
+    if (is_alp) {
```

Comment:
this code is overly complicated, we don't need to use it, just use a library to read this kind of files and then compare from test data that value with the value read by the library, sort of 3 lines of code and useful for each product to install.
Example of use of the library is in `lib/YaST/workarounds.pm` using use `Config::Tiny;`